README.md

```diff
@@ -150,6 +150,31 @@ public class SecurityConfiguration {
 
 [Full Stack Reactive with Spring WebFlux, WebSockets, and React](https://developer.okta.com/blog/2018/09/25/spring-webflux-websockets-react) uses both SSO and a resource server. Its current code uses Spring Security's OIDC support. [Changing it to use the Okta Spring Starter](https://github.com/oktadeveloper/okta-spring-webflux-react-example/pull/11) reduces the lines of code quite a bit.
 
+### Spring MVC
+
+To configure a resource server when using Spring MVC, you need to extend the `WebSecurityConfigurerAdapter` bean and override the `configure` method.
+
+```java
+@Configuration
+public class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapter {
```

Comment:
This is a Resource Server example, I would expect something for a more traditional MVC application in this example (similar to the example above)

Reply:
updated.